app/src/main/java/com/asfoundation/wallet/billing/adyen/AdyenPaymentFragment.kt

```diff
@@ -368,7 +373,8 @@ class AdyenPaymentFragment : DaggerFragment(), AdyenPaymentView {
   override fun showProductPrice(amount: String, currencyCode: String) {
     var fiatText = "$amount $currencyCode"
     if (isSubscription && frequency != null) {
-      frequency?.mapToSubFrequency(context, amount, currency)
+      val period = Period.parse(frequency!!)
+      period?.mapToSubCurrencyFrequency(context!!, fiatText)
```

Comment:
use `Subs` instead on `Sub` in the `mapToSubCurrencyFrequency`  method name to avoid ambiguous method name